internal/command/migrate_to_v2/volumes.go

```diff
@@ -53,9 +57,11 @@ func (m *v2PlatformMigrator) validateVolumes(ctx context.Context) error {
 }
 
 func (m *v2PlatformMigrator) migrateAppVolumes(ctx context.Context) error {
-
 	m.appConfig.SetMounts(lo.Map(m.appConfig.Mounts, func(v appconfig.Mount, _ int) appconfig.Mount {
 		v.Source = nomadVolNameToV2VolName(v.Source)
+		if len(v.Processes) == 0 {
+			v.Processes = m.appConfig.ProcessNames()
+		}
```

Comment:
If processes is set after `nomadVolNameToV2VolName`, I think the check in that function would return the wrong volume. Swapping the order of these operations should fix it

Reply:
good point!

Reply:
hmm now it is confusing, the `nomadVolNameToV2VolName` doesn't depend on the process group, it is quite simple actually:

```go
func nomadVolNameToV2VolName(name string) string {
  return name + forkedVolSuffix
}
```

Reply:
oh i'm tired sorry! you're right